### PR TITLE
Added exp fn

### PR DESCRIPTION
**Pull request by TVFlash**
Hi

### CODE REVIEW

**Comment by mattmolo** [comment]
This code is really bad Tim...

File: calc.py [hunk 1]
```diff
@@ -14,6 +14,9 @@ def div(a, b):
 def mod(a, b):
 	return a % b
 
+def exp(a, b):
```
Comment (mattmolo) [inline calc.py]: Why didn't you spell it out all the way?

File: calc.py [hunk 2]
```diff
@@ -14,6 +14,9 @@ def div(a, b):
...
+	return Math.pow(a, b)
```
Comment (mattmolo) [inline calc.py]: what about `return a**b`

**Comment by TVFlash** [comment]
Thanks matt 🤔 Merging anyway

**Pull request closed without merging**